GDASApp compute node build fails with develop at ab8fadd

Attempts to build GDASApp `develop` at [ab8fadd](https://github.com/NOAA-EMC/GDASApp/commit/ab8fadd714fb891cc016a2112af131da94829951) inside g-w `develop` at [8c7d6e6](https://github.com/NOAA-EMC/global-workflow/commit/8c7d6e6f36319b65b35c78601b0d12e36a085748) on compute nodes fail on multiple machines.  Below is the traceback from a GDASApp compute node build on Hera.
```
Processing /scratch3/NCEPDEV/da/Russ.Treadon/git/global-workflow/stable-nightly/sorc/gdas.cd/sorc/jcb
  Installing build dependencies: started
  Installing build dependencies: still running...
  Installing build dependencies: still running...
  Installing build dependencies: still running...
  Installing build dependencies: still running...
  Installing build dependencies: still running...
  Installing build dependencies: still running...
  Installing build dependencies: finished with status 'error'
  error: subprocess-exited-with-error

  ? pip subprocess to install build dependencies did not run successfully.
  ? exit code: 1
  ??> [10 lines of output]
      WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x147b64d982d0>: Failed to establish a new connection: [Errno 101] Network is unreachable')': /simple/setuptools/
      ERROR: Could not find a version that satisfies the requirement setuptools (from versions: none)
      ERROR: No matching distribution found for setuptools

      [notice] A new release of pip is available: 23.1.2 -> 25.3
      [notice] To update, run: python3.11 -m pip install --upgrade pip
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

? pip subprocess to install build dependencies did not run successfully.
? exit code: 1
??> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

[notice] A new release of pip is available: 23.1.2 -> 25.3
[notice] To update, run: python3.11 -m pip install --upgrade pip
CMake Error at CMakeLists.txt:136 (message):
  Failed to install jcb Python package
```

--- a/bundle/CMakeLists.txt
+++ b/bundle/CMakeLists.txt
@@ -128,7 +128,7 @@ if(BUILD_GDASBUNDLE)
 # Install jcb (JEDI Configuration Builder) Python package
   message(STATUS "Installing jcb (JEDI Configuration Builder)")
   execute_process(
-    COMMAND ${Python3_EXECUTABLE} -m pip install --no-deps --prefix ${CMAKE_INSTALL_PREFIX} .
+    COMMAND ${Python3_EXECUTABLE} -m pip install --no-deps --no-build-isolation --prefix ${CMAKE_INSTALL_PREFIX} .
     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/../sorc/jcb
     RESULT_VARIABLE jcb_install_result
   )

